Libs/MRML/Core/Testing/cube-utf8.mrml

```diff
@@ -1,4 +1,43 @@
-<MRML>
-<Camera
- id="vtkMRMLCameraNode1" name="camera(攝像頭)"  hideFromEditors="true"  position="1.26247 0.628757 -0.668912" focalPoint="0 0 0" viewUp="-0.566328 0.729902 -0.382773" parallelProjection="false" parallelScale="0" active="true"></Camera>
-</MRML>
+<?xml version="1.0" encoding="utf-8"?>
+<MRML>
+
+    <Camera
+        id="vtkMRMLCameraNode1"
+        name="camera(ae=æ AE=Æ a:=ä A:=Ä o|=ø O|=Ø o:=ö O:=Ö ao=å Ao=Å)"
```

Comment:
Is this what was intended?

Reply:
Yes. Does it look like an accident? It is a couple of Nordic characters with how it should approximately look like (using ASCII characters) and the real Unicode character.

Reply:
Right - I wasn't sure if this was on purpose.  It looked like something that might have come from some autoconvert, but now that you explain it sounds like a good test.